samples/Azure.ResourceManager.Sample/Generated/VirtualMachineExtensionContainer.cs

```diff
@@ -318,59 +316,21 @@ public async virtual Task<bool> DoesExistAsync(string vmExtensionName, string ex
             }
         }
 
-        /// <summary> The operation to get all extensions of a Virtual Machine. </summary>
-        /// <param name="expand"> The expand expression to apply on the operation. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<IReadOnlyList<VirtualMachineExtension>>> ListAsync(string expand = null, CancellationToken cancellationToken = default)
-        {
-            using var scope = _clientDiagnostics.CreateScope("VirtualMachineExtensionContainer.List");
-            scope.Start();
-            try
-            {
-                var response = await _restClient.ListAsync(Id.ResourceGroupName, Id.Name, expand, cancellationToken).ConfigureAwait(false);
-                return Response.FromValue(response.Value.Value.Select(data => new VirtualMachineExtension(Parent, data)).ToArray() as IReadOnlyList<VirtualMachineExtension>, response.GetRawResponse());
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
```

Comment:
Why this method got removed from the container class and moved to the `VirtualMachineExtensionOperations.cs` class? This method should be generated here.